content/docs/terraform-tools.mdx

```diff
@@ -0,0 +1,70 @@
+---
+page_title: Terraform Tools
+description: >-
+  Use these tools with Terraform to solve a broad range of infrastructure and DevOps challenges.
+
+---
+
+# Terraform Tools
+Terraform is part of a rich infrastructure and DevOps ecosystem. The tools below extend Terraform’s functionality or pair with Terraform to solve a broad range of infrastructure challenges.
+
+## Develop and Validate Configuration
+
+|Name | Maintainer |Description| Tutorial |
+|-----|------------|-----------|----------|
```

Comment:
Is there a way for us to make the columns more consistent (by combining columns or something like that)? The columns aren't evenly spaced and imo makes the table more difficult to skim ([Preview link](https://terraform-website-git-add-terraform-tools-hashicorp.vercel.app/docs/terraform-tools))

<details>
  <summary>screenshot showing misaligned tables</summary>

![image](https://user-images.githubusercontent.com/7556325/146493228-aec0103f-2a3d-46ca-82a3-0b9df02a4e34.png)

</details>

Reply:
for example, something like this is clean and easier to skim

![image](https://user-images.githubusercontent.com/7556325/146493780-aa232cdb-e9da-4d95-8f8a-5a28ba9ec431.png)

Reply:
I hear you - that does also bother me a bit BUT I still think that the tables make things easier to parse within each section. AND the names line up all together on the left side, so even though I don't LOVE that the table columns are not equal width, I think there is still value in the tables so I'm going to leave them. Thank you for calling this out though - I may talk to web dev about this in the future - this won't be the first time we want to use multiple tables on a page :)